lib/irc/IrcServer.js

```diff
@@ -68,6 +68,10 @@ IrcServer.prototype.getIdleTimeoutMs = function() {
     return this.config.ircClients.idleTimeout;
 };
 
+IrcServer.prototype.getReconnectInterval = function() {
```

Comment:
This is exactly why units are important :D Ideally the config file would've had that from the get-go rather than just `idleTimeout`.